build.gradle

```diff
@@ -47,7 +47,8 @@ plugins {
   // Updating the shadow plugin version to 8.1.1 causes issue with signing and publishing the shadowed
   // artifacts - see https://github.com/johnrengelman/shadow/issues/901
   id 'com.github.johnrengelman.shadow' version '8.1.0' apply false
-  id 'com.diffplug.spotless' version '6.14.0' apply false // 6.14.1 and newer require Java 11 at compile time, so we can't upgrade until AK 4.0
+  //  6.14.0 and newer are need for java 11
```

Comment:
how about: `the minimum required JRE of 6.14.0+ is 11`

Reply:
I refer to spotless github:   https://github.com/diffplug/spotless/tree/main/plugin-gradle#requirements

![image](https://github.com/apache/kafka/assets/48286053/582b0aa8-8912-419d-be81-8a869b9a2f2f)